src/main/java/com/cgdp/library/customValidator/ISBNConstraint.java

```diff
@@ -0,0 +1,13 @@
+package com.cgdp.library.customValidator;
+
+import javax.validation.Constraint;
+import java.lang.annotation.*;
+
+@Documented
+@Constraint(validatedBy = ISBNValidator.class)
+@Target({ElementType.FIELD, ElementType.PARAMETER})
+@Retention(RetentionPolicy.RUNTIME)
+public @interface ISBNConstraint {
+
```

Comment:
We should have a `required` property to control if this field is required or not, let's make it false by default
```suggestion
boolean isRequired() default false;
```